packages/execution-environments/src/common/enums.ts

```diff
@@ -2,3 +2,8 @@ export enum SNAP_STREAM_NAMES {
   JSON_RPC = 'jsonRpc',
   COMMAND = 'command',
 }
+
+export enum HandlerType {
+  onRpcRequest = 'onRpcRequest',
+  getTransactionInsight = 'getTransactionInsight',
+}
```

Comment:
```suggestion
export enum HandlerType {
  OnRpcRequest = 'onRpcRequest',
  GetTransactionInsight = 'getTransactionInsight',
}
```

Reply:
🤔 https://github.com/MetaMask/snaps-skunkworks/pull/642#discussion_r925246088

Reply:
See [TypeScript naming conventions](https://www.typescriptlang.org/docs/handbook/enums.html).

Reply:
Oh my bad, let's do `CamelCase.CamelCase` then per the official naming conventions.